File: matriz_python/Principal.py
import Matriz

lin = 10
col = 10
matriz = [[0 for j in range(col)] for i in range(lin)]
lista = []

matriz_obj = Matriz.Matriz(matriz, lin, col, lista)
matriz_obj.inicializar_matriz(matriz, lin, col)

matriz[0][3] = 1
matriz[2][5] = 1
matriz[5][7] = 1
matriz[6][8] = 1

matriz_obj.exibir_matriz(matriz, lin, col)

#converter matriz em lista
lista = []
matriz_obj.converter_matriz_lista(matriz, lin, col, lista)
matriz_obj.exibir_lista(lista)

File: matriz_java/Principal.java
import java.util.ArrayList;
public class Principal {
    public static void main(String[] args) {
        //simulando matriz especial
        int dimensao = 10;
        int matriz[][] = new int[dimensao][dimensao];
        Matriz.inicializarMatriz(matriz, dimensao, dimensao);

        matriz[0][3] = 1;
        matriz[2][5] = 1;
        matriz[5][7] = 1;
        matriz[6][8] = 1;
        Matriz.exibirMatriz(matriz, dimensao, dimensão);

        //converter matriz em lista de dados
        ArrayList<Dado> lista = new ArrayList<>();
        Matriz.converterMatrizLista(matriz, dimensao, dimensao, lista);
        Matriz.exibirLista(lista);
    }
}